src/icespeak/settings.py

```diff
@@ -216,3 +229,10 @@ def __eq__(self, other: object):
             "Could not load Google API key, ASR with Google will not work. Error: %s",
             err,
         )
+    try:
+        API_KEYS.openai = OpenAIKey.model_validate_json((_kd / SETTINGS.OPENAI_KEY_FILENAME).read_text().strip())
```

Comment:
Could you add fetching the OpenAI API key from the environment variable `OPENAI_API_KEY` as fallback if the json file doesn't exist? It is the only service out of these that usually uses an environment variable instead of a key file.